src/main/java/uk/gov/pay/connector/gateway/stripe/StripeFullTestCardNumbers.java

```diff
@@ -8,8 +8,12 @@ public class StripeFullTestCardNumbers {
 
     private static final String STRIPE_LOSING_EVIDENCE_CARD_NUMBER = "4000000000000259";
     private static final String STRIPE_WINNING_EVIDENCE_CARD_NUMBER = "4000000000002685";
+    private static final String SKIP_SUBMITTING_EVIDENCE_CARDHOLDER_NAME = "skip_evidence";
 
-    public static Optional<String> getSubmitTestDisputeEvidenceText(String firstSixDigits, String lastFourDigits) {
+    public static Optional<String> getSubmitTestDisputeEvidenceText(String firstSixDigits, String lastFourDigits, String cardholderName) {
+        if (cardholderName.equals(SKIP_SUBMITTING_EVIDENCE_CARDHOLDER_NAME)) {
```

Comment:
cardholder name can be null sometimes (if ledger is yet to process `PAYMENT_DETAILS_ENTERED` event) 
```suggestion
        if (SKIP_SUBMITTING_EVIDENCE_CARDHOLDER_NAME.equals(cardholderName) {
```

Reply:
Good spot, updated